Tools/CISupport/ews-app/ews/common/github.py

```diff
@@ -205,9 +205,10 @@ class GitHubEWS(GitHub):
                           ['webkitperl', 'ios-wk2', 'api-mac', 'gtk-wk2', ''],
                           ['webkitpy', 'api-ios', 'mac-wk1', 'api-gtk', ''],
                           ['jsc', 'tv', 'mac-wk2', 'jsc-armv7', ''],
-                          ['services', 'tv-sim', 'mac-AS-debug-wk2', 'jsc-armv7-tests', ''],
-                          ['merge', 'watch', 'mac-wk2-stress', 'jsc-mips', ''],
-                          ['unsafe-merge', 'watch-sim', '', 'jsc-mips-tests', '']]
+                          ['jsc-arm64', 'tv-sim', 'mac-AS-debug-wk2', 'jsc-armv7-tests', ''],
+                          ['services', 'watch', 'mac-wk2-stress', 'jsc-mips', ''],
+                          ['merge', 'watch-sim', '', 'jsc-mips-tests', ''],
+                          ['unsafe-merge', '', '', '', '']]
```

Comment:
This will be an empty row in most of the cases (unless there is unsafe-merge queue bubble present), but I guess that's ok.